assets/scripts/app/keyboard_commands.js

```diff
@@ -110,24 +112,40 @@ function getHoveredEl () {
   return el
 }
 
-// In case anyone tries a save shortcut key out of reflex,
-// we inform the user that it's not necessary.
-registerKeypress('ctrl s', {
-  trackAction: 'Command-S or Ctrl-S save shortcut key pressed'
-}, function () {
-  showStatusMessage(msg('STATUS_NO_NEED_TO_SAVE'))
-})
+export function registerKeypresses () {
+  // In case anyone tries a save shortcut key out of reflex,
+  // we inform the user that it's not necessary.
+  registerKeypress('ctrl s', {
+    trackAction: 'Command-S or Ctrl-S save shortcut key pressed'
+  }, function () {
+    showStatusMessage(msg('STATUS_NO_NEED_TO_SAVE'))
+  })
 
-// Catch-all for the Ctrl-S shortcut from ever trying to
-// save the page contents
-registerKeypress('ctrl s', {
-  preventDefault: true,
-  requireFocusOnBody: false
-}, _.noop)
+  // Catch-all for the Ctrl-S shortcut from ever trying to
+  // save the page contents
+  registerKeypress('ctrl s', {
+    preventDefault: true,
+    requireFocusOnBody: false
+  }, _.noop)
 
-// Catch-all for the backspace or delete buttons to prevent
-// browsers from going back in history
-registerKeypress(['backspace', 'delete'], {
-  preventDefault: true,
-  requireFocusOnBody: true
-}, _.noop)
+  // Catch-all for the backspace or delete buttons to prevent
+  // browsers from going back in history
+  registerKeypress(['backspace', 'delete'], {
+    preventDefault: true,
+    requireFocusOnBody: true
+  }, _.noop)
+
+  // Register keyboard input for show (shift-D)
+  registerKeypress('shift d', showDebugInfo)
```

Comment:
Yeah, you make good points here. I moved these registrations back to their own modules.